packages/phrases/src/locales/zh-cn/translation/demo-app.ts

```diff
@@ -1,5 +1,6 @@
 const demo_app = {
-  notification: '使用管理控制台帐号登录示例应用',
+  notification:
+    '管理控制台的用户名和密码是 demo app 的默认登录方式。点击下方创建账号或用现有账号登录。',
```

Comment:
I like the Chinese translated version. Can we align the English version to this?